contracts/distribution/dao-rewards-distributor/src/helpers.rs

```diff
@@ -130,13 +130,13 @@ impl DurationExt for Duration {
         }
     }
 
-    fn checked_div(&self, denominator: &Self) -> Result<Uint128, ContractError> {
+    fn ratio(&self, denominator: &Self) -> Result<Decimal, ContractError> {
```

Comment:
i like the idea of using decimals here. took a sec to understand what the return value is but lgtm